region/brinstar/kraid.json

```diff
@@ -1043,6 +1114,34 @@
                       ]
                     }
                   ]
+                },
+                {
+                  "name": "Tank the Damage",
+                  "notable": false,
+                  "requires": [
+                    {"enemyDamage": {
+                      "enemy": "Green Space Pirate (standing)",
+                      "type": "contact",
+                      "hits": 1
+                    }},
+                    {"enemyDamage": {
+                      "enemy": "Mini-Kraid",
+                      "type": "spike",
+                      "hits": 1
+                    }}
+                  ],
+                  "note": "Quickly run through to tank the Pirate (or its laser) and one of Mini-Kraid's spikes."
+                },
+                {
+                  "name": "Dodge",
+                  "notable": false,
+                  "requires": [],
```

Comment:
Since we didn't have a no-reqs path through this room before, should we leave this as `never` until we figure something out, rather than leaving it free?